src/lib/browse/layers/LayerControl.svelte

```diff
@@ -0,0 +1,91 @@
+<script lang="ts">
+  import { HelpButton } from "lib/common";
+  import { SecondaryButton, Checkbox, CheckboxGroup } from "govuk-svelte";
+  import { onDestroy } from "svelte";
+  import { activeLayers, layerLegends } from "./stores";
+
+  // Slots include: icon, controls, help
+
+  export let name: string;
+  export let title: string;
+  // This must be bound to state that controls the layer display and URL
+  export let show: boolean;
+
+  // If the layer is initially shown (from the URL state), make it active
+  if (show) {
+    $activeLayers.add(name);
+    $activeLayers = $activeLayers;
+  }
+
+  // The controls for each layer are defined in a layer component's optional
```

Comment:
This is a trick I've been using in many other projects. One Svelte component logically grouping some functionality can define some controls that wind up in different parts of the page. In this case, we're going a bit beyond that by centrally collecting all of the controls by name.